cmd/minishift/cmd/start_preflight.go

```diff
@@ -59,8 +61,20 @@ func preflightChecksBeforeStartingHost() {
 	case "hyperv":
 		preflightCheckSucceedsOrFails(
 			configCmd.SkipCheckHyperVDriver.Name,
-			checkHypervDriver,
-			"Checking if Hyper-V driver is configured",
+			checkHypervDriverInstalled,
+			"Checking if Hyper-V driver is installed",
+			false, configCmd.WarnCheckHyperVDriver.Name,
+			driverErrorMessage)
+		preflightCheckSucceedsOrFails(
+			configCmd.SkipCheckHyperVDriver.Name,
+			checkHypervDriverSwitch,
+			"Checking if Hyper-V driver is configured with Virtual Switch",
```

Comment:
Forgot to mention, I think it should be more explicit.   ` ... External Virtual Switch`. See https://docs.openshift.org/latest/minishift/getting-started/setting-up-driver-plugin.html#hyper-v-driver